watertap/unit_models/electrodialysis_0D.py

```diff
@@ -1546,6 +1716,179 @@ def eq_dl_thickness_ioa(self, mem, side, t):
                     * self.current_dens_lim_ioa[t] ** -1
                 )
 
+    def _get_fluid_dimensionless_quantities(self):
+        self.diffus_mass = Var(
+            initialize=1e-9,
+            bounds=(1e-16, 1e-6),
+            units=pyunits.meter**2 * pyunits.second**-1,
+            doc="The mass diffusivity of the solute as molecules (not individual ions)",
+        )
+        self.hydraulic_diameter = Var(initialize=1e-3, units=pyunits.meter)
+        self.N_Re = Var(
+            initialize=50,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Reynolds Number",
+        )
+        self.N_Sc = Var(
+            initialize=2000,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Schmidt Number",
+        )
+        self.N_Sh = Var(
+            initialize=100,
+            bounds=(0, None),
+            units=pyunits.dimensionless,
+            doc="Sherwood Number",
+        )
+
+        if self.config.hydraulic_diameter_method == HydraulicDiameterMethod.fixed:
+            _log.warning("Do not forget to FIX the channel hydraulic diameter in [m]!")
```

Comment:
Should this be an `.is_fixed()` assertion instead?

Reply:
I tried to do is_fixed() instead of _log.warning(), but it did not work very well. It turns out that the variables are generated without values, if you assert is_fixed in this step, it will be false automatically. The fixing step comes after this step, so a log warning is probably a better choice.

Reply:
Yeah, this has to be fixed at the flowsheet level so it's probably the best to leave as-is.  Thanks for testing.